common/data_guide.md

```diff
@@ -69,24 +57,24 @@ For the appropriate beneficiaries, the data provided to the ACO through BCDA inc
 ## Sample BCDA Files
 In order to aid in users’ understanding of BCDA file data and structure, attached are sample files for BCDA Explanation of Benefit, Patient, and Coverage files. These files contain synthetic data, but the structure of the files is similar to the actual files you will be pulling and downloading from BCDA.
```

Comment:
2nd clause of 1st sentence currently reads: attached are sample files for BCDA Explanation of Benefit, Patient, and Coverage files.

Should read: we provide sample Explanation of Benefit, Patient, and Coverage files for download.